node/tessel-export.js

```diff
@@ -569,24 +564,27 @@ Tessel.Pin.prototype.addListener = function(mode, callback) {
 
     // Can't set multiple listeners when using 'low' or 'high'
     if (this.interruptMode) {
-      if ((mode === 'low' || mode === 'high') || (this.listeners('low')[0] || this.listeners('high')[0])) {
+      var singleEventModes = ['low', 'high'];
+      if (singleEventModes.some(value => mode === value || this.interruptMode === value)) {
         throw new Error(`Cannot set pin interrupt mode to ${mode}; already listening for ${this.interruptMode}. Can only set multiple listeners with change, rise & fall`);
       }
     }
 
     // Set the socket reference so the script doesn't exit
     this._port.ref();
     this._setInterruptMode(mode);
-  }
 
-  // Add the event listener
-  Tessel.Pin.super_.prototype.on.call(this, mode, callback);
+    // Add the event listener
+    Tessel.Pin.super_.prototype.on.call(this, mode, callback);
+  } else {
+    throw new Error(`Invalid pin event mode '${mode}'. Valid modes are 'change', 'rise', 'fall', 'high' and 'low'.`);
```

Comment:
Use double-quotes inside of strings (per convention: https://github.com/dudleyjosh/t2-firmware/blob/jd-change-rise-fall-events/node/tessel-export.js#L562)

```js
Valid modes are "change", "rise", "fall", "high" and "low".
```